test/infrastructure/docker/Makefile

```diff
@@ -48,10 +48,18 @@ ifneq ($(abspath $(ROOT_DIR)),$(shell go env GOPATH)/src/sigs.k8s.io/cluster-api
 endif
 
 # Binaries.
-CONTROLLER_GEN := $(abspath $(TOOLS_BIN_DIR)/controller-gen)
-CONVERSION_GEN := $(abspath $(TOOLS_BIN_DIR)/conversion-gen)
-GOTESTSUM := $(abspath $(TOOLS_BIN_DIR)/gotestsum)
-KUSTOMIZE := $(abspath $(TOOLS_BIN_DIR)/kustomize)
+CONTROLLER_GEN_VER := v0.8.0
```

Comment:
Can we reuse the tools from top-level? (to avoid duplicating the version)

(same in test/e2e/Makefile)

Reply:
I have rewritten it to export the versions in the top-level but  I am not sure if that would suffice.

Reply:
I added a shell command to grep the version from top-level Makefile.